app/controllers/media_objects_controller.rb

```diff
@@ -17,11 +17,24 @@
 class MediaObjectsController < ApplicationController 
   include Avalon::Workflow::WorkflowControllerBehavior
   include Avalon::Controller::ControllerBehavior
+  include ConditionalPartials
 
 #  before_filter :enforce_access_controls
   before_filter :inject_workflow_steps, only: [:edit, :update]
   before_filter :load_player_context, only: [:show, :show_progress]
 
```

Comment:
Can the shared `if` proc be assigned to a lambda and reused below, both for clarity and maintainability in case the rules change?